phoenix-core/src/main/java/org/apache/phoenix/cache/ServerCacheClient.java

```diff
@@ -349,7 +350,7 @@ public TaskExecutionMetricsHolder getTaskExecutionMetric() {
                 }
             }
         }
-        if (LOG.isDebugEnabled()) {LOG.debug(addCustomAnnotations("Cache " + cacheId + " successfully added to servers.", connection));}
```

Comment:
`cacheId` is a byte array which as-is generates useless logs like `Cache [B@dcf3e99 successfully added to servers`